cookbook/wandb_logged_chain/wandb_chain.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from dotenv import load_dotenv
+from utils import get_time_in_ms
+from wandb_prompts.coolnessprompt import Coolness, CoolnessPrompt
+from wandb_prompts.partyinviteprompt import PartyInvitePrompt
+from wandb_prompts.whoprompt import Person, WhoPrompt
+
+import wandb
+from mirascope import OpenAIChat
+from wandb.sdk.data_types.trace_tree import Trace
+
+load_dotenv()
+
+if __name__ == "__main__":
+    wandb.login(key=os.getenv("WANDB_API_KEY"))
+    wandb.init(project="wandb_logged_chain")
+
+    root_span = Trace(
+        name="root",
+        kind="chain",
+        start_time_ms=get_time_in_ms(),
+        metadata={"user": "mirascope_user"},
+    )
+    chat = OpenAIChat(api_key=os.getenv("OPENAI_API_KEY"), model="gpt-3.5-turbo-1106")
+
+    who_prompt = WhoPrompt(span_type="tool", person="Brian")
+    start_time = get_time_in_ms()
+    who_completion = chat.extract(Person, who_prompt)
+    who_span = who_prompt.span(who_completion, root_span, start_time)
```

Comment:
completion already has start time

Reply:
they do now, so please update the examples to use completion start time and end time